src/test/groovy/com/avioconsulting/mule/linter/rule/property/PropertyFileNamingRuleTest.groovy

```diff
@@ -0,0 +1,28 @@
+package com.avioconsulting.mule.linter.rule.property
+
+import com.avioconsulting.mule.linter.model.Application
+import com.avioconsulting.mule.linter.model.Rule
+import com.avioconsulting.mule.linter.model.RuleViolation
+import spock.lang.Specification
+
+class PropertyFileNamingRuleTest extends Specification {
+
+    @SuppressWarnings(['MethodName', 'MethodReturnTypeRequired'])
+    def 'Property File Naming Rule check'() {
+        given:
+        Rule rule = new PropertyFileNamingRule(['dev', 'test', 'prod'])
+
+        when:
+        File appDir = new File(this.class.classLoader.getResource(application).file)
+        Application app = new Application(appDir)
+        List<RuleViolation> violations = rule.execute(app)
+
+        then:
+        violations.size() == size
+
+        where:
+        application     | size
+        'SampleMuleApp' | 0
+    }
+
+}
```

Comment:
This passes but isn't valid, because the class under test never returns any violations.  This test should be failing right now since you don't have a .prod properties file, but its passing anyway.

Reply:
this ticket was to just get and print the property file names, it wasn't the actual implementation of this rule...